bims/models/boundary.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
+"""Boundary model definition.
+"""
+
+from django.contrib.gis.db import models
+from bims.models.boundary_type import BoundaryType
+
+
+class Boundary(models.Model):
+    """Boundary model."""
+
+    name = models.CharField(
+        max_length=128,
+        blank=False,
+    )
+    type = models.ForeignKey(
+        BoundaryType, on_delete=models.CASCADE
+    )
+    geometry = models.MultiPolygonField(blank=True, null=True)
```

Comment:
I think we should have the scale range associated with each boundary (or perhaps as a property of BoundaryType) so that we can specify a scale range when requesting clusters.